### DISCUSSION TITLE
Start time tracking using the data from last entry

### DISCUSSION THREAD

**Discussion by FluffyDiscord** [solidtime-io/solidtime#243] [Feature requests]
Whenever I go to have lunch I have to stop the Time Tracker and when I come back I need to start it again. I would very much appreciate, if it would remember the previous data I entered (name/description, tags, projects, billable) when I start timer again. New button for it would be the best solution, I would guess.

**Comment by Eiernase** [comment]
This is already possible by clicking a time entry in the recent list.
Was this maybe just not closed?
